### PR TITLE
om-440

### PR DESCRIPTION
**Pull request by sidhartha-citron**

### CODE REVIEW

**Review by MHigel — COMMENTED** [review]
@sidhartha-citron I'm seeing a lot of changes here unrelated to the requirements in OM-440.  Can you verify that all of the items being removed from this Workflow are correct?

**Comment by sidhartha-citron** [comment]
Added all the workflowrules for Opportunity. Before i was adding only the new Workflow rule which got created. Seems it is overriding and removing other rules which it should be appending not removing. Now i have added all the rules.

File: force-app/main/default/workflows/Opportunity.workflow-meta.xml [hunk 1]
```diff
@@ -191,21 +202,70 @@
         <targetObject>AccountId</targetObject>
     </fieldUpdates>
     <fieldUpdates>
-        <fullName>Update_Account_Type</fullName>
-        <field>Type</field>
-        <literalValue>Customer</literalValue>
-        <name>Update Account Type</name>
+        <fullName>Update_Op_Name</fullName>
+        <field>Name</field>
+        <formula>Account.Name &amp; &quot; - &quot; &amp;  TEXT(Opportunity_Type__c) &amp; &quot; - &quot; &amp; TEXT(CloseDate)</formula>
```
Comment (MHigel) [inline force-app/main/default/workflows/Opportunity.workflow-meta.xml:207]: This commit will revert changes done to the formula that incorporates the `Subsidiary__c` value:

```<formula>AND(   ISPICKVAL(StageName, &apos;Signed&apos;) ,   ISPICKVAL( Processing_Status__c ,&apos;Completed&apos;),ISPICKVAL( Account.Subsidiary__c  , &apos;CA&apos;) )</formula>```

**Review by MHigel — CHANGES_REQUESTED** [review]

File: force-app/main/default/workflows/Opportunity.workflow-meta.xml (hunk 1 above)
Comment (sidhartha-citron) [reply force-app/main/default/workflows/Opportunity.workflow-meta.xml]: This is fixed now. Added Update Account Type Workflow Rule

**Review by MHigel — APPROVED** [review]

**Pull request merged**